neard/src/runtime.rs

```diff
@@ -743,6 +743,14 @@ impl RuntimeAdapter for NightshadeRuntime {
         Ok(header.signature().verify(header.hash().as_ref(), &block_producer.public_key))
     }
 
+    fn partial_verify_orphan_header_signature(&self, header: &BlockHeader) -> Result<bool, Error> {
```

Comment:
why do we need to add this new method? It seems that using `RuntimeAdapter::get_epoch_block_producers_ordered` would suffice.

Reply:
Changed to `get_block_producer` + `get_validator_by_account_id`